### ISSUE TITLE
NullPointerException in ChooseAccountDialogFragment

### ISSUE THREAD

**Issue by pavkamlc** [nextcloud/android#8413]
```
************ CAUSE OF ERROR ************

java.lang.NullPointerException
	at com.nextcloud.ui.ChooseAccountDialogFragment.getBinding(ChooseAccountDialogFragment.kt:71)
	at com.nextcloud.ui.ChooseAccountDialogFragment.avatarGenerated(ChooseAccountDialogFragment.kt:196)
	at com.owncloud.android.datamodel.ThumbnailsCacheManager$AvatarGenerationTask.onPostExecute(ThumbnailsCacheManager.java:921)
	at com.owncloud.android.datamodel.ThumbnailsCacheManager$AvatarGenerationTask.onPostExecute(ThumbnailsCacheManager.java:868)
	at android.os.AsyncTask.finish(AsyncTask.java:755)
	at android.os.AsyncTask.access$900(AsyncTask.java:192)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:772)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7397)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30160090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: xiaomi
Device: daisy_sprout
Model: Mi A2 Lite
Id: QKQ1.191002.002
Product: daisy

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V11.0.18.0.QDLMIXM
```

### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by stefan-niedermann** [comment]
Please fill out the complete issue template, especially the used versions and clear steps about how to reproduce the issue.

**Comment by AndyScherzinger** [comment]
Fixed via #8356
